--- /home/gray/texi2html-1.64/texi2html	Wed Jul 23 11:21:59 2003
+++ /home/gray/tmp/texi2html	Thu Jul 31 11:29:58 2003
@@ -3064,7 +3064,11 @@ INPUT_LINE: while ($_ = &next_line) {
 	my $after = $';
 	my @args;
 	my $args;
-	if ($after =~ /^\s*{(.*?[^\\])}(.*)/)  
+	if ($after =~ /^\s*{}(.*)/)
+	{
+	  $after = $1;
+	}
+	elsif ($after =~ /^\s*{(.*?[^\\])}(.*)/)  
 	{
 	  $args = $1;
 	  $after = $2;
@@ -3076,10 +3080,7 @@ INPUT_LINE: while ($_ = &next_line) {
 	  $args =~ s/\s*$//;
 	  $after = '';
 	}
-        else
-        {
-          $after = '';
-        }
+
 	$args =~ s|\\\\|\\|g;
 	$args =~ s|\\{|{|g;
         $args =~ s|\\}|}|g;
