--- /home/gray/texi2html-1.64/texi2html	Wed Jul 23 11:21:59 2003
+++ /usr/local/bin/texi2html	Fri Jul 25 13:11:13 2003
@@ -3069,6 +3069,10 @@ INPUT_LINE: while ($_ = &next_line) {
 	  $args = $1;
 	  $after = $2;
 	}
+	elsif ($after =~ /^\s*{}(.*)/)
+	{
+	  $after = $1;
+	}
 	elsif (@{$macros->{$name}->{Args}} == 1)
 	{
 	  $args = $after;
@@ -3076,10 +3080,7 @@ INPUT_LINE: while ($_ = &next_line) {
 	  $args =~ s/\s*$//;
 	  $after = '';
 	}
-        else
-        {
-          $after = '';
-        }
+
 	$args =~ s|\\\\|\\|g;
 	$args =~ s|\\{|{|g;
         $args =~ s|\\}|}|g;
